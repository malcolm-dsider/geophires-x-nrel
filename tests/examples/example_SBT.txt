************reservoir*************
Reservoir Model, 8
Reservoir Depth, 7.5 kilometer
Gradient 1, 60
Reservoir Volume Option, 4
Reservoir Volume, 8136407202.64, pi * 1 mi * 1 mi * 1000
Reservoir Heat Capacity, 1112
Reservoir Density, 2663
Reservoir Thermal Conductivity, 2.5

Lateral Endpoint Depth, 7.5 kilometer
Lateral Inclination Angle, 30.5
Junction Depth, 4.2 kilometer
Vertical Section Length, 4.2 kilometer
Number of Multilateral Sections, 12
SBT Accuracy Desired, 5
Lateral Spacing, 75
Discretization Length, 250
SBT Initial Timestep Count, 5
SBT Initial to Final Timestep Transition, 10000, seconds = one year
SBT Final Timestep Count, 120, ~4 per year

****************Wellbore***********
Is AGS, True
Well Geometry Configuration, 5
Number of Production Wells, 1
Number of Injection Wells, 1
Production Well Diameter, 8.5
Injection Well Diameter, 8.5
Nonvertical Wellbore Diameter, 0.216
Production Flow Rate per Well, 80
Reservoir Impedance, 1E-4, there is very little impedance in an AGS system
Multilaterals Cased, False

****************Surface plant************
End-Use Option, 1
Power Plant Type, 2
Plant Lifetime, 30
Ambient Temperature, 10
Surface Temperature, 10
Injection Temperature, 60

***************Economics****************
Economic Model, 3
Reservoir Stimulation Capital Cost, 0
Exploration Capital Cost, 0
Starting Electricity Sale Price, 0.19
Ending Electricity Sale Price, 0.19
xStarting Electricity Sale Price, 0.1
xEnding Electricity Sale Price, 100.0
xElectricity Escalation Start Year, 2
xElectricity Escalation Rate Per Year, 0.006956828, 3% inflation per year for 30 years (ends up at $0.35/kWh after 30 years)
xProduction Tax Credit Electricity, 0.04, PTC from IRA - can't have both so try one or the other
xInvestment Tax Credit Rate,0.5, ITC from IRA - can't have both so try one or the other
