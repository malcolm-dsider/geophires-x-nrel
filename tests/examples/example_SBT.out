                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.4.29
 Simulation Date: 2024-08-13
 Simulation Time:  11:57
 Calculation Time:     19.548 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Electricity
      Average Net Electricity Production:                     9.62 MW
      Electricity breakeven price:                           18.24 cents/kWh
      Number of production wells:                             1
      Number of injection wells:                              1
      Flowrate per production well:                          80.0 kg/sec
      Well depth:                                             0.0 kilometer
      Geothermal gradient:                                    0.0600 degC/m


                           ***ECONOMIC PARAMETERS***

      Economic Model  = BICYCLE
      Accrued financing during construction:                  0.00
      Project lifetime:                                      30 yr
      Capacity factor:                                       90.0 %
      Project NPV:                                           0.93 MUSD
      Project IRR:                                           6.31 %
      Project VIR=PI=PIR:                                    1.01
      Project MOIC:                                          0.65
      Project Payback Period:                               13.59 yr

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             1
      Number of Injection Wells:                              1
      Well depth:                                             0.0 kilometer
      Water loss rate:                                        0.0
      Pump efficiency:                                       75.0
      Injection temperature:                                 56.9 degC
      Production Wellbore heat transmission calculated with Ramey's model
      Average production well temperature drop:               0.0 degC
      Flowrate per production well:                          80.0 kg/sec
      Injection well casing ID:                               8.500 in
      Production well casing ID:                              8.500 in
      Number of times redrilling:                             0
      Power plant type:                                       Supercritical ORC


                         ***RESOURCE CHARACTERISTICS***

      Maximum reservoir temperature:                        400.0 degC
      Number of segments:                                     1
      Geothermal gradient:                                    0.0600 degC/m


                           ***RESERVOIR PARAMETERS***

The AGS models contain an intrinsic reservoir model that doesn't expose values that can be used in extensive reporting.


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Production Temperature:                       313.6 degC
      Average Production Temperature:                       218.7 degC
      Minimum Production Temperature:                       205.9 degC
      Initial Production Temperature:                       313.6 degC
The AGS models contain an intrinsic reservoir model that doesn't expose values that can be used in extensive reporting.


                          ***CAPITAL COSTS (M$)***

         Drilling and completion costs:                     104.87 MUSD
             Drilling and completion costs per vertical production well ( 4200 meter):         6.19 MUSD
             Drilling and completion costs per vertical injection well ( 4200 meter):          6.19 MUSD
             Drilling and completion costs for lateral sections (91919 meter):                92.48 MUSD
         Stimulation costs:                                   0.00 MUSD
         Surface power plant costs:                          52.76 MUSD
         Field gathering system costs:                        0.98 MUSD
         Total surface equipment costs:                      53.74 MUSD
         Exploration costs:                                   0.00 MUSD
      Total capital costs:                                  158.60 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

         Wellfield maintenance costs:                         1.44 MUSD/yr
         Power plant maintenance costs:                       1.93 MUSD/yr
         Water costs:                                         0.00 MUSD/yr
      Total operating and maintenance costs:                  3.37 MUSD/yr


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Initial geofluid availability:                          0.46 MW/(kg/s)
      Maximum Total Electricity Generation:                  17.60 MW
      Average Total Electricity Generation:                   9.62 MW
      Minimum Total Electricity Generation:                   8.40 MW
      Initial Total Electricity Generation:                  17.60 MW
      Maximum Net Electricity Generation:                    17.60 MW
      Average Net Electricity Generation:                     9.62 MW
      Minimum Net Electricity Generation:                     8.40 MW
      Initial Net Electricity Generation:                    17.60 MW
      Average Annual Total Electricity Generation:           75.03 GWh
      Average Annual Net Electricity Generation:             75.02 GWh
      Initial pumping power/net installed power:              0.01 %
      Average Pumping Power:                                  0.00 MW

                            ************************************************************
                            *  HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                            ************************************************************
  YEAR       THERMAL               GEOFLUID               PUMP               NET               FIRST LAW
             DRAWDOWN             TEMPERATURE             POWER             POWER              EFFICIENCY
                                     (degC)               (MW)              (MW)                  (%)
   1           1.0000                313.57               0.0010           17.5994               22.0597
   2           0.7703                241.55               0.0010           11.9572               20.8327
   3           0.7466                234.12               0.0009           11.2157               20.3593
   4           0.7332                229.89               0.0009           10.7910               20.0672
   5           0.7237                226.92               0.0009           10.4920               19.8524
   6           0.7163                224.62               0.0009           10.2606               19.6811
   7           0.7104                222.76               0.0009           10.0743               19.5399
   8           0.7054                221.20               0.0009            9.9172               19.4185
   9           0.7012                219.86               0.0009            9.7835               19.3136
  10           0.6975                218.71               0.0009            9.6684               19.2220
  11           0.6942                217.67               0.0009            9.5644               19.1382
  12           0.6911                216.72               0.0009            9.4692               19.0608
  13           0.6883                215.83               0.0009            9.3804               18.9878
  14           0.6856                214.98               0.0009            9.2958               18.9177
  15           0.6832                214.23               0.0009            9.2213               18.8554
  16           0.6809                213.52               0.0009            9.1505               18.7958
  17           0.6788                212.85               0.0009            9.0848               18.7400
  18           0.6768                212.22               0.0009            9.0220               18.6864
  19           0.6748                211.60               0.0009            8.9604               18.6334
  20           0.6729                211.01               0.0009            8.9022               18.5831
  21           0.6712                210.46               0.0009            8.8482               18.5361
  22           0.6695                209.94               0.0009            8.7965               18.4909
  23           0.6679                209.43               0.0009            8.7464               18.4468
  24           0.6663                208.92               0.0009            8.6966               18.4028
  25           0.6647                208.42               0.0009            8.6469               18.3586
  26           0.6631                207.94               0.0009            8.5996               18.3164
  27           0.6617                207.50               0.0009            8.5561               18.2774
  28           0.6603                207.05               0.0009            8.5127               18.2383
  29           0.6590                206.64               0.0009            8.4727               18.2020
  30           0.6577                206.24               0.0009            8.4329               18.1659


                              *******************************************************************
                              *  ANNUAL HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                              *******************************************************************
  YEAR             ELECTRICITY                   HEAT                RESERVOIR            PERCENTAGE OF
                    PROVIDED                   EXTRACTED            HEAT CONTENT        TOTAL HEAT MINED
                   (GWh/year)                  (GWh/year)            (10^15 J)                 (%)
   1                 133.2                       606.9                 83.01                   2.56
   2                  91.0                       442.5                 81.42                   4.43
   3                  86.7                       428.8                 79.88                   6.25
   4                  83.8                       420.1                 78.36                   8.02
   5                  81.8                       413.7                 76.88                   9.77
   6                  80.1                       408.7                 75.40                  11.50
   7                  78.8                       404.5                 73.95                  13.21
   8                  77.6                       401.0                 72.50                  14.90
   9                  76.7                       397.9                 71.07                  16.58
  10                  75.8                       395.3                 69.65                  18.25
  11                  75.0                       392.8                 68.23                  19.91
  12                  74.3                       390.6                 66.83                  21.56
  13                  73.6                       388.4                 65.43                  23.20
  14                  73.0                       386.5                 64.04                  24.84
  15                  72.4                       384.7                 62.65                  26.46
  16                  71.9                       383.0                 61.28                  28.08
  17                  71.4                       381.4                 59.90                  29.69
  18                  70.9                       379.9                 58.53                  31.30
  19                  70.4                       378.4                 57.17                  32.90
  20                  70.0                       377.0                 55.82                  34.49
  21                  69.6                       375.7                 54.46                  36.08
  22                  69.2                       374.4                 53.11                  37.66
  23                  68.8                       373.2                 51.77                  39.23
  24                  68.4                       372.0                 50.43                  40.81
  25                  68.0                       370.7                 49.10                  42.37
  26                  67.6                       369.6                 47.77                  43.93
  27                  67.3                       368.5                 46.44                  45.49
  28                  67.0                       367.5                 45.12                  47.04
  29                  66.6                       366.5                 43.80                  48.59
  30                  49.8                       274.2                 42.81                  49.75


                             ********************************
                             *  REVENUE & CASHFLOW PROFILE  *
                             ********************************
Year            Electricity             |            Heat                  |           Cooling                 |         Carbon                    |          Project
Since     Price   Ann. Rev.  Cumm. Rev. |   Price   Ann. Rev.   Cumm. Rev. |  Price   Ann. Rev.   Cumm. Rev.   |   Price   Ann. Rev.   Cumm. Rev.  | OPEX    Net Rev.      Net Cashflow
Start    (cents/kWh)(MUSD/yr) (MUSD)    |(cents/kWh) (MUSD/yr)    (MUSD)   |(cents/kWh) (MUSD/yr)    (MUSD)    |(USD/tonne) (MUSD/yr)    (MUSD)    |(MUSD/yr) (MUSD/yr)    (MUSD)
________________________________________________________________________________________________________________________________________________________________________________________
  1      0.00          -158.60   0.00     |    0.00     0.00         0.00    |    0.00     0.00         0.00     |    0.00     0.00         0.00     |  0.00     -158.60     -158.60
  2     19.00          21.93  25.31     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     21.93     -136.67
  3     19.00          13.93  42.61     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     13.93     -122.75
  4     19.00          13.09  59.07     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     13.09     -109.66
  5     19.00          12.56  75.00     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     12.56     -97.10
  6     19.00          12.16  90.54     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     12.16     -84.94
  7     19.00          11.85  105.76     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     11.85     -73.08
  8     19.00          11.60  120.73     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     11.60     -61.49
  9     19.00          11.38  135.49     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     11.38     -50.11
 10     19.00          11.19  150.05     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     11.19     -38.91
 11     19.00          11.03  164.46     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     11.03     -27.88
 12     19.00          10.88  178.71     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     10.88     -17.00
 13     19.00          10.74  192.83     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     10.74     -6.26
 14     19.00          10.61  206.81     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     10.61      4.35
 15     19.00          10.50  220.68     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     10.50     14.85
 16     19.00          10.39  234.44     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     10.39     25.24
 17     19.00          10.28  248.10     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     10.28     35.52
 18     19.00          10.19  261.66     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     10.19     45.71
 19     19.00          10.09  275.13     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     10.09     55.80
 20     19.00          10.00  288.51     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37     10.00     65.81
 21     19.00           9.92  301.80     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37      9.92     75.73
 22     19.00           9.84  315.02     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37      9.84     85.57
 23     19.00           9.77  328.16     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37      9.77     95.33
 24     19.00           9.69  341.22     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37      9.69     105.02
 25     19.00           9.62  354.21     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37      9.62     114.64
 26     19.00           9.54  367.13     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37      9.54     124.18
 27     19.00           9.48  379.98     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37      9.48     133.66
 28     19.00           9.41  392.76     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37      9.41     143.07
 29     19.00           9.35  405.48     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37      9.35     152.42
 30     19.00           9.29  418.14     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  3.37      9.29     161.71
